src/store/network/networkActions.js

```diff
@@ -6,100 +6,99 @@ import createLogger from '../../utils/logger';
 import ActionTypes from './actionTypes';
 import history from '../wallet/history';
 
+import network from './';
```

Comment:
this would cause a circular dependency?

Reply:
yes,  but webpack doesnt seem to care...

Reply:
really hard to make this work without this.

Reply:
you have these actions in this file?

Reply:
yes, but they are wrapped in the index.js 

I could wrap them within this file, and call the wrapped variant instead I suppose